### PR TITLE
Added QA Quiz Question

### PR DESCRIPTION
**Pull request by shikharcodes**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a [descriptive title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title) (**not** a vague title like `Update index.md`)

Adds to #102

The added question covers regression tests.

### CODE REVIEW

**Comment by shikharcodes** [comment]
Please add the "hacktoberfest-accepted" label before merging this, as I've contributed this under hacktoberfest 2022.

File: src/data/quality-assurance-quiz.ts [hunk 1]
```diff
@@ -86,6 +86,18 @@ const qualityAssuranceQuiz = [
       "Even if the app is 99% bug free, if it doesn't meet consumer requirements or fulfill the original user requirements it is useless.",
     Link: "https://www.freecodecamp.org/news/software-testing-beginners-guide/",
   },
+  {
+    Question:
+      "What do you mean by regression tests?",
```
Comment (jdwilkin4) [inline src/data/quality-assurance-quiz.ts:91]:
```suggestion
      "What are regression tests?",
```

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]
Thank you for your contribution!

Please review the requested changes.
Once those changes have been implemented, then we can merge your PR into main. 👍

**Review by jdwilkin4 — APPROVED** [review]
Congrats on your first contribution to our repository. 🎉
We look forward to future contributions. 👍

**Pull request merged**